.ci/azure/mac.yml

```diff
@@ -1,48 +1,61 @@
 jobs:
 - job: Mac
   # About 200% of total time (perfomace of Mac hosts is unstable)
-  timeoutInMinutes: 180
+  timeoutInMinutes: 240
```

Comment:
maybe it's time to add?

 `trigger:
- master`

Reply:
not master only, on all commits, it is configured in Azure